template/9-design.md

```diff
@@ -13,10 +24,17 @@
 ## Data Model
 
 *What data are you collecting / managing?*
+- User: id, name, email, password
+- Post: id, user_id, content, location, created_at
+- Comment: id, user_id, content, created_at
+- Vote: id, user_id, post_id, value, created_at
+- Challenge: id, title, description, start_date, end_date, reward
```

Comment:
I wonder what you'd think of reorganising this to follow a little more the firestore structure ?
I don't know if this is very relevant though.

```suggestion
- User: id, name, email, password, joined date
  - UserComments: comment_id, content, post_id
- Post: id, user_id, content, location, created_at
  - Vote : id, user_id, vote_type
  - Comment: id, user_id, content, created_at
    - Vote : id, user_id, vote_type
- Challenge: id, title, description, start_date, end_date, reward
```

Reply:
fixed in https://github.com/ProximaEPFL/prd/pull/1/commits/aca5a2afcd725a5e54dcb78fe234b8ce21cc5276